inductiva/examples/examples.py

```diff
@@ -0,0 +1,102 @@
+"""Functions to download and load examples from URLs.
+
+It contains several samples of molecules and a test_vehicle."""
+import os
+from typing import Optional
+from urllib import request, error
+
+from absl import logging
+
+import inductiva
+
+
+def load_insulin(save_dir: Optional[str] = None) -> str:
+    """Load the insulin molecule.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1ZNI", save_dir)
+
+
+def load_hemoglobin(save_dir: Optional[str] = None) -> str:
+    """Load the hemoglobin molecule.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1A3N", save_dir)
+
+
+def load_lysozyme(save_dir: Optional[str] = None) -> str:
+    """Load lysozyme protein.
+    Returns:
+        The path to the downloaded PDB file."""
+    return download_from_rcsb("1AKI", save_dir)
+
+
+def load_test_vehicle(save_dir: Optional[str] = None) -> str:
+    """Load a test vehicle from Inductiva Github.
+
+    Returns:
+        The path to the downloaded OBJ file."""
+
+    return download_inductiva_resources("test_vehicle.obj", save_dir)
```

Comment:
Same comment. Why not just use download_inductiva_resources 
It's perfect and self-explanatory!

Team: please don't make things more complicated than they need to be....!